tests/Sniffs/Arrays/data/sorted.fixed.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+[ /* inline comment */ // opening line comment
+	'foo',
```

Comment:
I think that there's should be no fix (and maybe no error) when some values are without key, see https://3v4l.org/UaZGu

Reply:
Works for me...
With the new ArrayHelper class it will be trivial to create a sniff to enforce all values are keyed if any are.
What should that sniff be called  (will be separate pull request)